golos.charge/golos.charge.abi

```diff
@@ -287,6 +325,14 @@
     "name": "use",
     "type": "use"
   },
+  {
+    "name": "usenotifygt",
+    "type": "useandnotify"
+  },
+  {
+    "name": "usenotifyls",
```

Comment:
I refer name `usenotifylt` - short form from use and notify less then. It's easy to call another method, for example: usenotifyge - great or equal. gt/ge lt/le more or less well-established form of contraction